Deprecate own kafka implementation, recommend use of new spring-kafka,

maihacke opened the pull request:
see https://github.com/devonfw/devon4j/issues/459

maybeec commented:
@maihacke could you please help with a short replace all on your PR? It was too much for me to do it by hand as there were too many files, which I can just edit by permissions within the github UI.
Please replace "Devon4Js Kafka module" with "devon4j-kafka" to not introduce non-proper spelling in all docs.

Btw. remains this PR to be WIP as stated in the headline?

maihacke commented:
> @maihacke could you please help with a short replace all on your PR? It was too much for me to do it by hand as there were too many files, which I can just edit by permissions within the github UI. Please replace "Devon4Js Kafka module" with "devon4j-kafka" to not introduce non-proper spelling in all docs.
> 
> Btw. remains this PR to be WIP as stated in the headline?

I replaced every occurence as requested:

`find . -type f -name '*.java' -exec sed -i '' 's/Devon4Js Kafka module/devon4j-kafka/g' {} +`

Removed WIP. I still hoping to find the time to update the [example app](https://github.com/devonfw-sample/devon4j-kafka-employeeapp) but that is another story.
